### PR TITLE
Do not run conda init during install

### PR DESCRIPTION
**Pull request by inducer**

### CODE REVIEW

**Comment by inducer** [comment]
cc @lukeolson

File: install.sh [hunk 1]
```diff
@@ -55,7 +55,6 @@ export MY_CONDA_DIR=$conda_prefix
 export PATH=$MY_CONDA_DIR/bin:$PATH
 
 echo "==== Create 'dgfem' conda environment"
-conda init
```
Comment (matthiasdiener) [inline install.sh:58]: The info at the bottom of this file probably needs to be adjusted as well.

File: install.sh [hunk 2]
```diff
@@ -55,7 +55,6 @@ export MY_CONDA_DIR=$conda_prefix
...
 
 # Attempt to also run conda init for the users shell (in case it is not bash):
 which finger 2>/dev/null >/dev/null && conda init $(finger $USER | grep 'Shell:*' | cut -f3)
```
Comment (matthiasdiener) [inline install.sh:60]: These lines also need to be remove then.

File: install.sh (hunk 1 above)
Comment (inducer) [reply install.sh:58]: Good catch, thanks!

**Comment by inducer** [comment]
I can't leave a positive review, but I much prefer this way of doing business over our previous way.

**Comment by inducer** [comment]
Also, from my perspective, this is ready to go in.

**Pull request merged**